cognite/neat/_utils/validation.py

```diff
@@ -1,23 +1,63 @@
+from collections.abc import Callable, Mapping
+from typing import Literal
+
 from pydantic import ValidationError
 from pydantic_core import ErrorDetails
 
 
-def humanize_validation_error(error: ValidationError) -> list[str]:
+def as_json_path(loc: tuple[str | int, ...]) -> str:
+    """Converts a location tuple to a JSON path.
+
+    Args:
+        loc: The location tuple to convert.
+
+    Returns:
+        A JSON path string.
+    """
+    if not loc:
+        return ""
+    # +1 to convert from 0-based to 1-based indexing
+    prefix = ""
+    if isinstance(loc[0], int):
+        prefix = "item "
```

Comment:
Why do you want a space after `item`?
I tried this: `as_json_path((1, "test", 3, "check", "some_prop"))`
And it returned: `'item [2].test[4].check.some_prop'`